templates/oauth2/oauth2-login.html.twig

```diff
@@ -0,0 +1,144 @@
+
+{#
+ # Authorization Server Member
+ #
+ # @package   OpenEMR
+ # @link      http://www.open-emr.org
+ # @author    Jerry Padgett <sjpadgett@gmail.com>
+ # @copyright Copyright (c) 2020 Jerry Padgett <sjpadgett@gmail.com>
+ # @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+#}
+{% extends "oauth2/oauth2-base.html.twig" %}
+{% block title %}{{ "OpenEMR Authorization"|xlt }}{% endblock %}
+{% block content %}
+<div class="row h-100 w-100 justify-content-center align-items-center">
+    <div class="col-sm-6 bg-light text-dark">
+        <div class="text-md-center">
+        {% if mfaRequired %}
+            <h4 class="mb-4 mt-1">{{'MFA Verification'|xlt }}</h4>
+        {% else %}
+            <h4 class="mb-4 mt-1">{{ "Sign In"|xlt  }}</h4>
+        {% endif %}
```

Comment:
Since the headers are the same for both cases, you could put the conditions inside the tag to control just the text that will be rendered. That way you avoid repeating the markup

Reply:
Fixed.